src/Components/Backoffice/Backoffice.js

```diff
@@ -0,0 +1,17 @@
+import CardBackoffice from "./Card/CardBackoffice"
+const Backoffice = ({ elementBackoffice}) => {
```

Comment:
element backoffice is not scope of this user story. it should only render a welcome message